src/analytics.js

```diff
@@ -1,23 +0,0 @@
-define([], function () {
```

Comment:
I'm kinda annoyed that this module had to be removed/replaced with one called `hqAnalytics` (it messes up git history of our analytics code). The rename seems unnecessary. Am I missing something important about why this had to be done?